cmd/gdeploy/commands/provider/provider.go

```diff
@@ -10,5 +14,35 @@ var Command = cli.Command{
 	Subcommands: []*cli.Command{
 		&addCommand, &removeCommand, &updateCommand,
 	},
-	Action: cli.ShowSubcommandHelp,
+	Action: cli.ActionFunc(func(ctx *cli.Context) error {
+		cli.ShowSubcommandHelp(ctx)
+		DeprecatedWarn(ctx)
+		return nil
+	}),
+}
+
+func DeprecatedWarn(c *cli.Context) error {
+	// add a deprecated warning here
+	clio.Warn("Warning: this command is deprecated and no longer supported")
```

Comment:
I think we should just redirect the user and explain the benefits. We will continue to support the add command but the detailed instructions will be available in the web UI.

```suggestion
	clio.Warn("Configuring access providers via gdeploy has been deprecated in favour of an interactive setup flow in your admin dashboard. Head there now get started setting up your provider.")
```